LocalPackages/SubscriptionUI/Package.swift

```diff
@@ -25,7 +24,6 @@ let package = Package(
                 .product(name: "BrowserServicesKit", package: "BrowserServicesKit"),
                 .product(name: "Subscription", package: "BrowserServicesKit"),
                 .product(name: "SwiftUIExtensions", package: "SwiftUIExtensions"),
-                .product(name: "PreferencesUI-macOS", package: "PreferencesUI-macOS"),
```

Comment:
@ayoy this is the issue, If I remove it works (and shouldn't) if I leave it everything dies